.github/workflows/ci.yml

```
@@ -29,6 +29,12 @@ jobs:
       - name: Install deps
         run: |
           apt update && apt install -y --no-install-recommends ca-certificates cmake build-essential clang llvm git pkg-config autoconf automake libtool libelf-dev wget libb64-dev libc-ares-dev libcurl4-openssl-dev libssl-dev libtbb-dev libjq-dev libjsoncpp-dev libgrpc++-dev protobuf-compiler-grpc libgtest-dev libprotobuf-dev liblua5.1-dev linux-headers-amd64
+          mkdir third_party && cd third_party
+          wget "https://github.com/tristanpenman/valijson/archive/refs/tags/v0.6.tar.gz"
+          tar xzf v0.6.tar.gz && cd valijson-0.6
+          mkdir -p build
+          cd build && cmake -Dvalijson_INSTALL_HEADERS=ON ../
+          make install
```

We might want to run this in a subshell so that the work directory gets reverted to what it was before we `cd`'ed into `build`
```suggestion
         (
            cd build && cmake -Dvalijson_INSTALL_HEADERS=ON ../
            make install
         )
```

---

The build instructions were moved to a dedicated file to avoid overcrowding the workflow file.